tests/foreman/api/test_errata.py

```diff
@@ -1365,37 +1364,75 @@ def _validate_swid_tags_installed(vm, module_name):
     assert module_name in result
 
 
+@pytest.fixture
+def errata_host_lce(module_sca_manifest_org, target_sat):
+    """Create and return a new lce in module SCA org."""
+    return target_sat.api.LifecycleEnvironment(organization=module_sca_manifest_org).create()
+
+
+def fetch_cv_by_name_and_cvv(sat, org, name=DEFAULT_CV, ver=None):
```

Comment:
Which test is calling this function?
Also could you please add `:param:` section which explain meaning of each parameter?
```suggestion
def fetch_cv_by_name_and_cvv(sat, org, name=DEFAULT_CV, ver=None):
    """ Doc string
    
    :param: <type> sat:
    :param: <type> org:
    :param: <type> name:
    :param: <type> ver:
     """
```

Reply:
Planned for next PR of a few enhancements, but is unneeded for now. 
Will remove from here, and put up w/ requested changes once fully implemented